src/cli/test/ionic-config.spec.ts

```diff
@@ -2,6 +2,10 @@ import { mockLogger } from '@stencil/core/testing';
 import { getCompilerSystem, initializeStencilCLIConfig } from '../state/stencil-cli-config';
 import { readConfig, writeConfig, updateConfig, defaultConfig } from '../ionic-config';
 import { createSystem } from '../../compiler/sys/stencil-sys';
+import { UUID_REGEX } from '../telemetry/helpers';
+
+const UUID1 = '5588e0f0-02b5-4afa-8194-5d8f78683b36';
+const UUID2 = 'e5609819-5c24-4fa2-8817-e05ca10b8cae';
```

Comment:
IIUC, this will still write back to `~/.ionic/config.json` right? Do we have a story on the books to have this write to a test file & have the suite clean up after itself after the face??

Reply:
STENCIL-53!